integration/tests/existing_vpc/existing_vpc_test.go

```diff
@@ -136,7 +136,7 @@ func createVPC(stackName string, ctl api.ClusterProvider) *api.ClusterVPC {
 	publicSubnets, privateSubnets, vpcID, securityGroup := createVPCStackAndGetOutputs(stackName, ctl)
 
 	By("creating the cluster config from the existing VPC")
-	newVPC := api.NewClusterVPC()
+	newVPC := api.NewClusterVPC(false)
```

Comment:
sorry, I'm blind. When is this ever true?

Reply:
its mainly used for testing. But if you wanted to generate a VPC config for ipv6 its needs to be true

Reply:
But how or where do you set that?

Reply:
Ahh! Found it. :D

```go
		if clusterConfig.VPC == nil {
			clusterConfig.VPC = api.NewClusterVPC(ipv6Enabled)
		}
```